mm/alignment.py

```diff
@@ -789,3 +792,160 @@ def plot(self):
             plt.xticks(rotation=45)
 
             plt.savefig(plot_file)
+
+
+class DumpSegmentTextAlignmentJob(Job):
+    """
+    Dumps all text and alignments for the given corpus and alignment files in a human-readable csv format.
+    """
+
+    def __init__(
+        self,
+        corpus_file: tk.Path,
+        alignment_files: Iterable[tk.Path],
```

Comment:
It is unclear which file format is expected. maybe the name `alignment_caches` would make it more clear that rasr caches are expected. otherwise add a bit more info to docstr pls.

Reply:
I just modified the variables to be `alignment_caches`. That makes sense.